examples/launcher-templates/default/2.3.0/job.tpl

```diff
@@ -0,0 +1,213 @@
+# Version: 2.3.0
```

Comment:
Are these templates landing in a separate PR?

Reply:
Yes, this PR is stacked on top of: https://github.com/rstudio/helm/pull/325

Reply:
GitHub isn't showing the PR as stacked; maybe change the base branch?

Reply:
Changed my base branch. I didn't realize GH would rebase automatically after my first PR was merged!